prime-router/settings/prod/0163-nv-prod-up-onboarding.yml

```diff
@@ -0,0 +1,109 @@
+- name: "nv-phd"
+  description: "Nevada Public Health Department"
+  jurisdiction: "STATE"
+  stateCode: "NV"
+  countyName: null
+  filters: []
+  featureFlags: null
+  keys: null
+  senders: []
+  receivers:    
+  - name: "full-elr"
+    organizationName: "nv-phd"
+    topic: "full-elr"
+    customerStatus: "inactive"
+    translation: !<HL7>
+      schemaName: "metadata/hl7_mapping/receivers/STLTs/NV/NV-receiver-transform"
+      useTestProcessingMode: false
+      useBatchHeaders: true
+      receivingApplicationName: "NVDOH"
+      receivingApplicationOID: null
+      receivingFacilityName: "NVODH"
+      receivingFacilityOID: null
+      messageProfileId: null
+      replaceValue: {}
+      replaceValueAwithB:
+        MSH-4-1:
+        - '*': "CDC PRIME"
+        MSH-4-2:
+        - '*': "11D2030855"
+      reportingFacilityName: null
+      reportingFacilityId: null
+      reportingFacilityIdType: null
+      suppressQstForAoe: false
+      suppressHl7Fields: null
+      suppressAoe: true
+      defaultAoeToUnknown: false
+      replaceUnicodeWithAscii: false
+      useBlankInsteadOfUnknown: null
+      truncateHDNamespaceIds: true
```

Comment:
Ahh I missed these other couple when I was looking before... I think this also needs to go in the transform?

Reply:
We do support this setting in the UP, but it does need to be tested.